data/json/items/armor/boots.json

```
@@ -267,7 +267,7 @@
     "description": "A pair of customized Kevlar boots, heavily armored with superalloy and modified to provide maximum protection from harm, even when knee-deep in the dead.",
     "material": [ "kevlar", "superalloy" ],
     "color": "light_cyan",
-    "encumbrance": 30,
+    "encumbrance": 27,
```

Retains them being 5 less encumbrance than the regular steel version
```suggestion
    "encumbrance": 25,
```